sky/provision/aws/config.py

```diff
@@ -565,17 +587,26 @@ def _get_or_create_vpc_security_group(ec2, vpc_id: str,
             VpcId=vpc_id,
         )
     except ec2.meta.client.exceptions.ClientError as e:
+        if e.response['Error']['Code'] == 'InvalidGroup.Duplicate':
+            # The security group already exists, but we didn't see it
+            # because of eventual consistency.
+            logger.warning(f'{expected_sg_name} already exists when creating.')
+            security_group = _get_security_groups(ec2, vpc_id, expected_sg_name)
+            assert (security_group is not None and
+                    security_group.group_name == expected_sg_name), \
+                f'Expected {expected_sg_name} but got {security_group}'
```

Comment:
```suggestion
            assert ((security_group is not None and
                    security_group.group_name == expected_sg_name),
                f'Expected {expected_sg_name} but got {security_group}')
```
nit: use parentheses for linebreak